### DISCUSSION TITLE
github copilot被封禁 无法退款

### DISCUSSION THREAD

**Discussion by ghost** [community/community#113445] [Copilot Conversations]
### Select Topic Area

Question

### Body

加入了恶意的组织 然后被封了权限 新开了一年 但是没法使用 目前也没说退款问题 怎么办

**Comment by ghostinhershell** [comment]
Hi @LianXiaoxiang, thanks for participating!

We take our [Code of Conduct](https://github.com/community/community/blob/0ca37cf594e438fd4aec3c7f3736b95e6b0b3567/CODE_OF_CONDUCT.md) very seriously and want to help ensure that everyone has a good experience free of antagonism and harassment. Unfortunately, we don’t currently have moderators for languages other than English. Until that changes, we need to ask that everyone use English here in the GitHub Community when posting. We’ll be locking any posts in languages other than English for now, including this one.
